<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>914</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des Clients</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton#start_game{
	
	background-color: rgb(255, 209, 25);
	border-color: rgb(104, 104, 104);
	color: rgb(104, 104, 104);
	font: 75 10pt &quot;8514oem&quot;;
	border-width: 5px;
}
QPushButton#start_game::hover{
	
	background-color: rgb(255, 209, 25);
	color: #c23b22;
	border-color: #c23b22;

	border-width: 5px;
	font: 75 25pt &quot;8514oem&quot;;
}
QLabel#QrCode{
background-color:#e6f2ed;
}

QLabel#player1_score_arduino{
	font: 75 18pt &quot;Myanmar Text&quot;;
	color: rgb(0, 0, 127);
	border-color: rgb(0, 0, 0);
	background-color: rgb(253, 177, 56);
    border-width: 5px;
	border-radius:10px;
}
QLabel#player2_score_aduino{
	font: 75 18pt &quot;Myanmar Text&quot;;
	color: rgb(0, 0, 127);
	background-color: rgb(193, 230, 60);
    border-width: 5px;
	border-radius:10px;
	border-color: rgb(0, 0, 0);

}
/*-----QWidget-----*/
QWidget
{
	
	background-color: #f2faf7;
	color: #ffffff;
	border-color: #222021;

}

/*-----QLabel-----*/

QLabel
{
	background-color: transparent;
	color: #424242;
	font: 12pt &quot;Corbel&quot;;
}


QLabel::disabled
{
	background-color: transparent;
	color: #656565;

}

QMenuBar
{
	background-color: #77bb78;

}

QMenuBar::disabled
{
	background-color: #404040;
	color: #656565;
	border-color: #051a39;

}


QMenuBar::item
{
    background-color: transparent;

}


QMenuBar::item:selected
{
    background-color: #68a365;
    border: 1px solid #41cd52;

}


QMenuBar::item:pressed
{
    background-color: #41cd52;
    border: 1px solid #000;
    margin-bottom: -1px;
    padding-bottom: 1px;

}

/*-----QToolTip-----*/
QToolTip
{
	border : 1px solid #000000;
	border-raduis:1 px;
	color: white;
	border-color: transparent;

}

QToolBox
{
	border : 1px solid #000000;
	background-color: rgb(194, 208, 168);
	color: white;
	border-color:transparent;
	border-raduis:3 px;


}
/*-----QToolButton-----*/
QToolButton
{
	background-color: #607cff;
	color: #ffffff;
	border-width: 1px;
	border-radius: 3px;
	border-color: #051a39;
	padding: 3px;

}


QToolButton::disabled
{
	background-color: #404040;
	color: #656565;
	border-color: #051a39;

}


QToolButton::hover
{
	background-color: #8399ff;
	color: #ffffff;
	border-width: 1px;
	border-radius: 3px;
	border-color: #051a39;
	padding: 3px;

}


QToolButton::pressed
{
	background-color: #4969ff;
	color: #ffffff;
	border-width: 1px;
	border-radius: 3px;
	border-color: #051a39;
	padding: 3px;

}

/*-----QPushButton-----*/
QPushButton
{	
	background-color:  #c23b22;
	border-style: solid;
	border-width: 2px;
	border-radius: 10px;
	border-color:  #ffffff;
	padding: 10px;
	font: 75 15pt &quot;Corbel&quot;;
	

}
/*#E37783B*/
QPushButton::pressed
{	
	background-color: #E37783;
	color:#f1f3f4 ;
	
border-style: solid;

    /*border-bottom-color: qlineargradient(spread:pad, x1:0, y1:1, x2:1, y2:1, stop:0 #100E19, stop:0.5 #100E19, stop:0.6 #C0DB50, stop:1 #C0DB50);*/

	border-width: 4px;
    border-radius: 3px;
	padding: 2px;

}
QPushButton::hover
{
	background-color: #f1f3f4;
	color: #c23b22;
	border-style: solid;
	border-width: 2px;
	border-radius: 10px;
	border-color: #c23b22; 
	padding: 10px;
	font: 75 15pt &quot;Corbel&quot;;
}

/*-----QLineEdit-----*/
QLineEdit
{
	color: #ffffff;
	background-color: rgb(209, 209, 209);
	border-width: 1px;
	border-radius: 3px;
	border-color: #051a39;
	padding: 2px;

}


QLineEdit::pressed
{
	background-color: #d9d9d9;
	color: #383838;
	border-width: 1px;
	border-radius: 3px;
	border-color: #4e4f4f;
	padding: 2px;

}


QLineEdit::disabled
{
	background-color: #404040;
	color: #656565;
	border-width: 1px;
	border-radius: 3px;
	border-color: #051a39;
	padding: 2px;

}


/*-----QTextEdit-----*/
QTextEdit
{
	background-color: rgb(209, 209, 209);
	color: #010201;
	border-color: transparent;
	border-radius: 5px;

}


QTextEdit::disabled
{
	background-color: #404040;
	color: #656565;
	border-color: #051a39;
	border-radius: 3px;


}


/*-----QGroupBox-----*/
QGroupBox 
{
    border: 1px solid;
  	border-radius: 10px;
	background-color:#f2faf7;/*f2faf7 aftah chwaya*/
	border-color: trasparent; 
    margin-top: 10px;

}

QGroupBox::title  
{
    
	background-color: rgb(213, 213, 88);
    color: #ffffff;
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 5px;
	border-top-left-radius: 3px;
	border-top-right-radius: 3px;

}


QGroupBox::title::disabled
{
	background-color: #404040;
	color: #656565;
    subcontrol-origin: margin;
    subcontrol-position: top left;
    padding: 5px;
	border-top-left-radius: 3px;
	border-top-right-radius: 3px;

}


/*-----QSpinBox &amp; QDoubleSpinBox &amp; QDateTimeEdit-----*/
QSpinBox, 
QDoubleSpinBox,
QDateTimeEdit
{
	
	background-color:  rgb(209, 209, 209);
	color: rgb(2, 2, 2);
	border-radius: 3px;

}


QSpinBox::disabled, 
QDoubleSpinBox::disabled,
QDateTimeEdit::disabled
{
	background-color: #404040;
	color: #656565;
	border-color: #051a39;

}


QSpinBox:hover, 
QDoubleSpinBox::hover,
QDateTimeEdit::hover
{
    background-color: #626262;
    border: 1px solid #c23b22;
    color:  #fff;
    padding: 2px;
}


QSpinBox::up-button, QSpinBox::down-button,
QDoubleSpinBox::up-button, QDoubleSpinBox::down-button,
QDateTimeEdit::up-button, QDateTimeEdit::down-button
{
	
	background-color: #c23b22;
	color: rgb(255, 255, 255);
	border-radius: 2px;
border:2px;
border-color:rgb(209, 209, 209);
}

QSpinBox::disabled, 
QDoubleSpinBox::disabled,
QDateTimeEdit::disabled
{
	background-color: #404040;
	color: #656565;
	border-color: #051a39;

}


QSpinBox::up-button:hover, QSpinBox::down-button:hover,
QDoubleSpinBox::up-button:hover, QDoubleSpinBox::down-button:hover,
QDateTimeEdit::up-button:hover, QDateTimeEdit::down-button:hover
{
    background-color: #ff7a52;
    border: 1px solid #ff7a52;
	color: rgb(255, 255, 255);

}


QSpinBox::up-button:disabled, QSpinBox::down-button:disabled,
QDoubleSpinBox::up-button:disabled, QDoubleSpinBox::down-button:disabled,
QDateTimeEdit::up-button:disabled, QDateTimeEdit::down-button:disabled
{
	background-color: #404040;
	color: #656565;
	border-color: #051a39;
	color: rgb(255, 255, 255);

}


QSpinBox::up-button:pressed, QSpinBox::down-button:pressed,
QDoubleSpinBox::up-button:pressed, QDoubleSpinBox::down-button::pressed,
QDateTimeEdit::up-button:pressed, QDateTimeEdit::down-button::pressed
{
    background-color: #de621f;
    border: 1px solid #de621f;

}


QSpinBox::down-arrow,
QDoubleSpinBox::down-arrow,
QDateTimeEdit::down-arrow
{
  	
    width: 7px;

}


QSpinBox::up-arrow,
QDoubleSpinBox::up-arrow,
QDateTimeEdit::up-arrow
{
    image: url(://arrow-up.png);
    width: 7px;

}


/*-----QCheckBox-----*/
QCheckBox::indicator
{
    color: #b1b1b1;
    background-color: #323232;
    border: 1px solid #b1b1b1;
    width: 12px;
    height: 12px;

}


QCheckBox::indicator:checked
{
    image:url(://checkbox.png);
    border: 1px solid #68a365;
}


QCheckBox::indicator:unchecked:hover
{
    border: 1px solid #68a365;

}


QCheckBox::disabled
{
	color: #656565;

}


QCheckBox::indicator:disabled
{
	background-color: #656565;
	color: #656565;
    border: 1px solid #656565;

}


/*-----QRadioButton-----*/
QRadioButton{
font: 10pt &quot;Corbel&quot;;
	color:#c23b22;
}
QRadioButton::indicator::unchecked
{ 
	border: 2px inset gray; 
	border-radius: 5px; 
	background-color:  #fff;
	width: 9px; 
	height: 9px; 
	font: 8pt &quot;Corbel&quot;;
	color:#c23b22;

}


QRadioButton::indicator::unchecked:hover
{ 
	border: 2px solid #c23b22; 
	border-radius: 5px; 
	background-color:  #fff;
	width: 9px; 
	height: 9px; 


}


QRadioButton::indicator::checked
{ 
	border: 2px inset darkgray; 
	border-radius: 5px; 
	background-color: #c23b22; 
	width: 9px; 
	height: 9px; 

}


QRadioButton::disabled
{
	color: #656565;

}


QRadioButton::indicator:disabled
{
	background-color: #656565;
	color: #656565;
    border: 2px solid #656565;

}

/*-----QTableView &amp; QTableWidget-----*/
QTableView
{
	background-color:#e6f2ed;
    border: transparent;
	color: rgb(0, 0, 0);
    gridline-color:#c23b22;
	border-radius: 10px;
border-color: transparent;
}
QTableView::item:hover 
{
    
	
	background-color: #c8e3d8;
    color: #c23b22;

}
QTableView::item:selected 
{
    background-color: #1a1b1c;
    border: 2px solid #68a365;
    color: #F0F0F0;

}

QTableView::item:selected:disabled
{
    background-color: #1a1b1c;
    border: 2px solid #525251;
    color: #656565;

}


QTableCornerButton::section
{
    background-color: #505050;
    color: #fff;

}


QHeaderView::section
{
    background-color: #525251;
    color: #fff;
    text-align: left;
	padding: 4px;
	
}


QHeaderView::section:disabled
{
    background-color: #525251;
    color: #656565;

}


QHeaderView::section:checked
{
    color: #fff;
    background-color: #68a365;

}


QHeaderView::section:checked:disabled
{
    color: #656565;
    background-color: #525251;

}


QHeaderView::section::vertical::first,
QHeaderView::section::vertical::only-one
{
    border-top: 1px solid #353635;

}


QHeaderView::section::vertical
{
    border-top: 1px solid #353635;

}


QHeaderView::section::horizontal::first,
QHeaderView::section::horizontal::only-one
{
    border-left: 1px solid #353635;

}


QHeaderView::section::horizontal
{
    border-left: 1px solid #353635;

}
/*-----QTabWidget-----*/

QTabWidget::background
{
	background-color: #777877;
	color: #f2faf7;
    border-color: transparent;
}
QTabBar::tab
{
	background-color:#5b5c5b ;
	color: #85ba72;
	border-style: solid;
	border-width: 1px;
	border-top-left-radius: 3px;
	border-top-right-radius: 3px;
	border-color: #051a39;
	padding: 5px;
	font: 8pt &quot;Corbel&quot;;

}


QTabBar::tab:disabled
{
	background-color: #5b5c5b;
	color: #656565;

}

QTabBar::tab:selected, QTabBar::tab:last:selected, QTabBar::tab:hover {
	
	background-color: #5b5c5b;
	color: rgb(255, 255, 255);
  	border-bottom-color: #04b97f;
	border-bottom-width: 2px;
}

QTabWidget::pane 
{
	background-color: transparent;
	color: rgb(248, 255, 97);
    border: 1px solid;
    border-color: transparent;
}



QTabBar::tab:selected
{
   
	background-color:#85ba72;
	color: #ffffff;
	border-style: solid;
	border-width: 1px;
	border-top-left-radius: 3px;
	border-top-right-radius: 3px;
	border-color: #051a39;
	padding: 5px;

}


QTabBar::tab:selected:disabled
{
	background-color: #404040;
	color: #656565;

}


QTabBar::tab:!selected 
{
    background-color: #4d4d4d;

}


QTabBar::tab:!selected:hover 
{
    background-color: #a2a3a2;
	color:#537851;

}


QTabBar::tab:top:!selected 
{
    margin-top: 3px;

}


QTabBar::tab:bottom:!selected 
{
    margin-bottom: 3px;

}


QTabBar::tab:top, QTabBar::tab:bottom 
{
    min-width: 8ex;
    margin-right: -1px;
    padding: 5px 10px 5px 10px;

}


QTabBar::tab:top:selected 
{
    border-bottom-color: none;

}


QTabBar::tab:bottom:selected 
{
    border-top-color: none;

}


QTabBar::tab:top:last, QTabBar::tab:bottom:last,
QTabBar::tab:top:only-one, QTabBar::tab:bottom:only-one 
{
    margin-right: 0;

}


QTabBar::tab:left:!selected 
{
    margin-right: 3px;

}


QTabBar::tab:right:!selected
{
    margin-left: 3px;

}


QTabBar::tab:left, QTabBar::tab:right 
{
    min-height: 8ex;
    margin-bottom: -1px;
    padding: 10px 5px 10px 5px;

}


QTabBar::tab:left:selected 
{
    border-left-color: none;

}


QTabBar::tab:right:selected 
{
    border-right-color: none;

}


QTabBar::tab:left:last, QTabBar::tab:right:last,
QTabBar::tab:left:only-one, QTabBar::tab:right:only-one 
{
    margin-bottom: 0;

}

/*-----QScrollBar-----*/
QScrollBar:horizontal
{
    border: 1px solid , #68a365;
    
	background-color: rgb(171, 141, 72);
    height: 13px;
    margin: 0px 16px 0 16px;

}


QScrollBar::handle:horizontal
{
    background: rgb(223, 209, 46);
    min-height: 20px;

}


QScrollBar::add-line:horizontal
{
    border: 1px solid #1b1b19;
   
	background-color: rgb(214, 214, 116);
    width: 14px;
    subcontrol-position: right;
    subcontrol-origin: margin;

}


QScrollBar::sub-line:horizontal
{
    border: 1px solid #1b1b19;
    background-color: rgb(214, 214, 116);
    width: 14px;
    subcontrol-position: left;
    subcontrol-origin: margin;

}


QScrollBar::right-arrow:horizontal
{
    image: url(://arrow-right.png);
    width: 6px;
    height: 6px;

}


QScrollBar::left-arrow:horizontal
{
    image: url(://arrow-left.png);
    width: 6px;
    height: 6px;

}


QScrollBar::add-page:horizontal, QScrollBar::sub-page:horizontal
{
    background: none;

}


QScrollBar:vertical
{
    background-color: #3d3d3d;
    width: 13px;
    margin: 16px 0 16px 0;
    border: 1px solid #222222;

}


QScrollBar::handle:vertical
{
    background-color: #607cff;
    min-height: 20px;

}


QScrollBar::add-line:vertical
{
    border: 1px solid #1b1b19;
    background-color: #607cff;
    height: 14px;
    subcontrol-position: bottom;
    subcontrol-origin: margin;

}


QScrollBar::sub-line:vertical
{
    border: 1px solid #1b1b19;
    background-color: #607cff;
    height: 14px;
    subcontrol-position: top;
    subcontrol-origin: margin;

}


QScrollBar::up-arrow:vertical
{
    image: url(://arrow-up.png);
    width: 6px;
    height: 6px;

}


QScrollBar::down-arrow:vertical
{
    image: url(://arrow-down.png);
    width: 6px;
    height: 6px;

}


QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical
{
    background: none;

}
</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>10</y>
      <width>841</width>
      <height>471</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QPushButton" name="pushButton_singIn">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>370</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Connecter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_singUp">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>370</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>s'inscrire</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>120</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>utilisateur</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>220</y>
        <width>101</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Mot de passe</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_username">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>120</y>
        <width>201</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_mdp">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>210</y>
        <width>201</width>
        <height>51</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_close">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>370</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Quitter</string>
      </property>
     </widget>
     <widget class="QLabel" name="logo">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>110</y>
        <width>181</width>
        <height>171</height>
       </rect>
      </property>
      <property name="text">
       <string>logo</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_3">
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>60</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>170</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Age</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>110</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Prénom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>320</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Téléphone</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>270</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>mot de passe</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>220</y>
        <width>161</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom d'utilisateur</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_NOM">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>70</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_PRENOM">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>120</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_AGE">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>170</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_USERNAME">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>220</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_MDP">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>270</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_TELEPHONE">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>320</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_enregistrer">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>390</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Enregistrer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_annuler_Enregistrement">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>390</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Annuler</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QPushButton" name="pb_gestion_materiels">
      <property name="geometry">
       <rect>
        <x>164</x>
        <y>30</y>
        <width>461</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Gestion Materiels</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_gestion_finance">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>100</y>
        <width>461</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Gestion Finance</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_gestion_personnel">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>170</y>
        <width>461</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Gestion Personnel</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_gestion_cours">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>240</y>
        <width>461</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Gestion cours</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_gestion_sponsors">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>310</y>
        <width>461</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Gestion Sponsors</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_gestion_adherents">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>380</y>
        <width>461</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Gestion d'adherents</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>914</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
